app-ui-catalog/src/main/java/app/k9mail/ui/catalog/CatalogActivity.kt

```diff
@@ -1,13 +1,13 @@
 package app.k9mail.ui.catalog
 
 import android.os.Bundle
-import androidx.activity.ComponentActivity
 import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import androidx.core.view.WindowCompat
+import androidx.fragment.app.FragmentActivity
 import app.k9mail.core.ui.compose.common.activity.setActivityContent
 import app.k9mail.ui.catalog.ui.CatalogScreen
 
-class CatalogActivity : ComponentActivity() {
+class CatalogActivity : FragmentActivity() {
```

Comment:
I personally don't like that we have to use the FragmentActivity, but the BiometricPrompt is bound to it. Hoping for an update in the future that relaxes this dependency.

Reply:
We need to update the `FeatureLauncherActivity` too.

Reply:
I'd defer this until the component is actually used by a feature launched from `FeatureLauncherActivity`.